src/containers/shared/components/Transaction/DIDSet/TableDetail.tsx

```diff
@@ -0,0 +1,27 @@
+import { useTranslation } from 'react-i18next'
+import { TransactionTableDetailProps } from '../types'
+import { DIDSet } from './types'
+import { convertHexToString } from '../../../../../rippled/lib/utils'
+
+export const TableDetail = ({
+  instructions,
+}: TransactionTableDetailProps<DIDSet>) => {
+  const { t } = useTranslation()
+  const { URI, DIDDocument } = instructions
+  return (
+    <div className="didSet">
+      {URI && (
+        <div className="uri">
+          <span className="label">{t('uri')}: </span>
+          <span className="lower">{convertHexToString(URI)}</span>
+        </div>
+      )}
+      {URI && (
+        <div className="did-document">
+          <span className="label">{t('did_document')}: </span>
+          <span className="lower">{convertHexToString(DIDDocument)}</span>
```

Comment:
Why is the class name `lower`?

Reply:
Short for "lowercase" - though I guess I really meant case-sensitive

Reply:
That is what i figured but it has the same styling as `.account` which isn't only lower case so I caught it.

Reply:
I can change it to `case-sensitive` or `no-upper`

Reply:
`case-sensitive` is my vote.